### PR TITLE
237: scrutinizer clean up

### PR DESCRIPTION
**Pull request by toolstack**
Additional cleanups based on scrutinizer, part of #237.

### CODE REVIEW

**Comment by GaryJones** [comment]
I'm not sure which coding standard you're using, but there are rules being updated that include whitespace and some that don't e.g.:

``` php
$rules->name_should_not_be('empty');
```

 vs

``` php
$rules->status_should_not_be( 'empty' );
```

Maybe not strictly picked up in Scrutinizer, but now would be the ideal time to fix them up for consistency.

**Comment by toolstack** [comment]
In those cases I was just replacing the variable names, the rest was left as originally done.  There's a case to be made to go through the entire codebase and fix up the coding layout.

**Comment by GaryJones** [comment]
> There's a case to be made to go through the entire codebase and fix up the coding layout.

If this project follows the same principles as WP core, then PRs just for the sake of code formatting is generally frowned upon, hence the case for doing this now when the lines of code are already being touched.

File: gp-includes/cli/translation-set.php [hunk 1]
```diff
@@ -7,7 +7,7 @@ class GP_CLI_Translation_Set extends WP_CLI_Command {
 	 * @param string $project Project path
 	 * @param string $locale Locale slug
 	 * @param string $set Set slug
-	 * @return GP_Translation_Set|WP_Error Translation set if available, error otherwise.
+	 * @return object Translation set (GP_Translation_Set class) if available, error (WP_Error class) otherwise.  Note, PHPDoc doesn't know about WP_Error so we can explicitly define it in the return types.
```
Comment (ocean90) [inline gp-includes/cli/translation-set.php]: I don't accept that reason. We don't write code for tools, we're writing code for users and developers.

Comment (toolstack) [inline gp-includes/cli/translation-set.php]: I'll revert it, but slightly disagree with the above.  Doc blocks serve a dual purpose, make it easy for developers to understand the code and let tools work with the codebase in a consistent manner.

The above change accomplishes both goals where as the original format only accomplished one of them.

Comment (ocean90) [inline gp-includes/cli/translation-set.php]: >  and let tools work with the codebase in a consistent manner.

True, but it's a plugin for WordPress. Maybe there is a way to include WordPress as a library? Another example is `WP_Query`.

Comment (toolstack) [inline gp-includes/cli/translation-set.php]: Not sure, other people must have run in to this, we should look in to it more.

Comment (ocean90) [inline gp-includes/cli/translation-set.php]: Do you have a link handy for the scrutinizer issue?

Comment (toolstack) [inline gp-includes/cli/translation-set.php]: Should be [these three](https://scrutinizer-ci.com/g/GlotPress/GlotPress-WP/issues/develop/files/gp-includes/cli/translation-set.php?orderField=path&order=asc&fileId=gp-includes%2Fcli%2Ftranslation-set.php&honorSelectedPaths=0) if I recall correctly

**Comment by ocean90** [comment]
Reviewing.

**Pull request merged**